tests/test_fitsRawFormatter.py

```diff
@@ -136,9 +136,15 @@ def setUp(self):
         self.warnContext = self.assertLogs(level="WARNING")
         self.logContext = lsst.log.UsePythonLogging()
 
+        # Make a data ID to pass to the formatter.
+        universe = lsst.daf.butler.DimensionUniverse()
+        dataId = lsst.daf.butler.DataCoordinate.standardize(instrument="Cam1", exposure=2, detector=10,
+                                                            physical_filter="Cam1-R",
+                                                            band="r", universe=universe)
```

Comment:
Correct.  I think if new code needs them, they can just switch `dataId` to `self.dataId`, but it's premature to do that now.

Also, I did make `detector=10` consistent with line 90, because I don't like leaving inconsistencies lurking in test data, and I didn't notice until now that there was also some metadata that used `u` as a filter-something, so I'll switch the filters used here, too.  I didn't see anything that would specify or otherwise constrain what the other made-up values should be.